### PR TITLE
Add demo submission guidelines

### PR DESCRIPTION
**Pull request by Calinou**
The pull request template was changed to match godotengine/godot's.

See https://github.com/godotengine/godot-demo-projects/issues/697#issuecomment-1509959276.

### CODE REVIEW

**Review by aaronfranke — APPROVED** [review]
Wonderful :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    then removing `.godot/` within the ZIP file.
+  - For 3D demos, consider relying on Godot's procedural generation abilities
+    (such as [NoiseTexture2D](https://docs.godotengine.org/en/stable/classes/class_noisetexture2d.html))
+    to decrease file size.
+
+### Submission criteria for new demos
+
+If you are submitting a new demo:
+
+- Make sure it includes a `READE.md` file similar to the other demos.
```
Comment (TheYellowArchitect) [inline CONTRIBUTING.md:46]: Typo at: `README.md`

Comment (Calinou) [reply CONTRIBUTING.md:46]: Feel free to open a PR to fix it :slightly_smiling_face:

Comment (Calinou) [reply CONTRIBUTING.md:46]: https://github.com/godotengine/godot-demo-projects/pull/933

**Pull request merged**